lib/selinux/module.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package selinux
+
+import (
+	"bytes"
+	_ "embed"
+	"text/template"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/lib/versioncontrol"
+)
+
+const (
+	moduleName = "teleport_ssh"
+	domain     = "teleport_ssh_t"
+)
+
+//go:embed teleport_ssh.te
+var module []byte
+
+//go:embed teleport_ssh.fc.tmpl
+var fileContexts string
```

Comment:
Under the assumption that many more files will be needed for other Teleport features... how much space will this approach end up using?

I am concerned that over time this will end up further bloating the teleport binary size as this project progresses.

Reply:
This file will likely never grow beyond 50 lines, struggle to see how it could get to 20 honestly. The `.te` file will be bigger, but even bigger policies like OpenSSH's are still under a 1000 lines, and that covers sshd and many client binaries